transforms/code/malware/python/src/malware_local_python.py

```diff
@@ -50,6 +51,6 @@
     # Set the simulated command line args
     sys.argv = ParamsUtils.dict_to_req(d=params | malware_params)
     # create launcher
-    launcher = PythonTransformLauncher(runtime_config=MalwareTransformConfiguration())
+    launcher = PythonTransformLauncher(runtime_config=MalwarePythonTransformConfiguration())
     # Launch the ray actor(s) to process the input
```

Comment:
I not it is not part of the PR, but ray actor(s) in the pure python implementation.

Reply:
It just fixing and obvious bug

Reply:
not my issue